### PR TITLE
Add environement variables

### PR DESCRIPTION
**Pull request by adnenre**
i created the .env file to remove hardcoded variables like http://localhost:9000 and the port 9000 
also package-lock.json does not need to be published

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+VITE_API_URL=http://localhost:9000
+VITE_APP_PORT=9000
```
Comment (niksumeiko) [inline .env]: How about not to couple these names to a bundler? Today vite, tomorrow webpack.

```suggestion
API_URL=http://localhost:9000
APP_PORT=9000
```

**Review by niksumeiko — APPROVED** [review]
Great addition. Thanks a lot, @adnenre. LGTM&zwnj;_!_ 👍

File: .env (hunk 1 above)
Comment (adnenre) [reply .env]:
```javascript
export default defineConfig(({ mode }) => {
    const env = loadEnv(mode, process.cwd(), '');
    return {
        plugins: [react()],
        server: {
            port: 5173,
        },
        test: {
            include: ["**/*.test.ts"],
        },
        define: {
            "process.env.API_URL": JSON.stringify(env.API_URL),

            // If you want to exposes all env variables, which is not recommended
            // 'process.env': env
        },
    };
});
```
the configuration file need to be changed then we can expose the custom env variable

**Review by adnenre — COMMENTED** [review]
done

**Review by niksumeiko — APPROVED** [review]
LGTM&zwnj;_!_

**Pull request merged**